amgut/db/patches/0041.sql

```diff
@@ -116,14 +116,13 @@ CREATE INDEX idx_sample_plate_layout_sample_id ON pm.sample_plate_layout ( sampl
 COMMENT ON COLUMN pm.sample_plate_layout.name IS 'The name of the sample in this plate in case that needs to be changed (e.g. if the sample has been plated twice)';
 
 CREATE TABLE pm.study (
-	study_id             bigserial  NOT NULL,
-	qiita_study_id       bigint  ,
-	title                varchar  ,
-	alias                varchar  ,
-	notes                varchar  ,
+	study_id             bigint   NOT NULL,
+	title                varchar  NOT NULL,
+	alias                varchar  NOT NULL,
+    jira_key			 varchar  NOT NULL,
```

Comment:
Note that in JIRA you need the internal project id to instantiate and, for example, delete a project, not the key.